### PR TITLE
Fix checking pad

### PR DESCRIPTION
**Pull request by primenumber**
## What this patch does to fix the issue.
correct order: `[top, bottom, left, right]`
(it is same as tf.pad)

### CODE REVIEW

File: tests/converter/test_consistency_check.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def test_conv_consistency(self) -> None:
             [1, 3, 3, 3],
             Float32(),
             input_ops,
-            pads=[1, 1, 2, 2],
+            pads=[1, 2, 1, 2],
```
Comment (iizukak) [inline tests/converter/test_consistency_check.py:117]: @primenumber 
Sorry, I can not understand this change clear.
pad format is `[top, bottom, left, right]` and this line means using different padding value for top and buttom. right?

Comment (iizukak) [inline tests/converter/test_consistency_check.py:117]: Sorry, I can not understand this change clearly.  
pad format is [top, bottom, left, right] and this line means using different padding value for top and buttom. right?

Comment (primenumber) [reply tests/converter/test_consistency_check.py:117]: Yes.

**Pull request merged**